eva/interfaces/relational/db.py

```diff
@@ -169,6 +199,40 @@ def load(
         stmt = parse_load(table_name, file_regex, format, **kwargs)
         return EVARelation(self._evadb, stmt)
 
+    def drop(self, item_name: str, item_type: str, **kwargs) -> EVARelation:
+        """
+        Drop a table or UDF from the database.
+
+        Args:
+            item_name (str): Name of the table or UDF to be dropped.
+            item_type (str): Type of item either: table or udf.
+            **kwargs: Additional keyword arguments for configuring the load operation.
+
+        Returns:
+            EVARelation: The EVARelation object representing the loaded table.
+        """
+        # DROP {item_type} item_name
+        stmt = parse_drop(item_name, item_type, **kwargs)
+        return EVARelation(stmt)
+
+    def create(self, item_type: str, item_name: str, **kwargs) -> EVARelation:
+        """
+        Create a table or UDF and add it to the database.
+
+        Args:
+            item_type (str): Type of item either: table or udf.
+            name (str): Name of the table or UDF to be created.
+            model_type (str): Type of model to be created could be 'classification', 'object-detection', etc.
+            model (str): Path of the model weigths.
+            **kwargs: Additional keyword arguments for configuring the load operation.
```

Comment:
Add support for `if_not_exists.` It is super important. Default set it to `True`